src/version.h

```diff
@@ -106,7 +106,7 @@ const char *GetVersionString();
 #if defined(__APPLE__) || defined(_WIN32)
 #define GAME_DIR GAMENAME
 #else
-#define GAME_DIR ".config/" GAMENAMELOWERCASE
+#define GAME_DIR FString(g_XDGconf + "/" + FString(GAMENAMELOWERCASE)) // g_XDGconf is defined in i_system.h and initialized in i_specialpaths.cpp
```

Comment:
What I am worried here is that this part of the code is not enclosed within the `defined(__unix__)` block like with the variable declaration in `i_system.h`, and then the `#else` block would be left as is. Perhaps the `#else` block should enclose `".config/" GAMENAMELOWERCASE` in `FString` parenthesis, so that it should still work with the other changes.